test/attr/attr_objc.swift

```diff
@@ -142,10 +142,10 @@ class subject_getterSetter1 {
   }
 
   var observingAccessorsVar1: Int = 0 {
-    @objc // expected-error {{observing accessors are not allowed to be marked @objc}} {{5-11=}}
+    @objc // expected-error {{willSet observer is not allowed to be marked @objc}} {{5-11=}}
```

Comment:
Should `willSet` and `didSet` be in quotes?

Reply:
this was an outdated message

here we have the new message
https://github.com/swiftlang/swift/pull/74462/commits/3cd17c788695c79488d5bf8dc49528e2c53da9d3

> observers (didSet observer) are not allowed to be marked '@objc'


`didSet observer` is `accessor->getDescriptiveKind())`

Reply:
I think we should still mark that with quotes maybe?

Reply:
> I think we should still mark that with quotes maybe?

We should, but I think we can leave this out for someone else to fix as a first contribution. The missing quotes in the descriptive decl kind string is an issue of its own:

https://github.com/swiftlang/swift/blob/fa72680d28380f81a755a2975ba3afb0aaef06f9/lib/AST/Decl.cpp#L370-L371

@iMostfa Up to you.